src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -76,7 +76,7 @@
     "d:transactions/measurements.time_to_full_display@millisecond": PREFIX + 127,
 }
 
-# 200 - 299
+# 200 - 399
```

Comment:
Since we're going to be adding more tag keys during experimentation, I took the liberty to increase this size.